packages/kotti-ui/source/kotti-field-inline-edit/components/ConfirmButton.vue

```diff
@@ -21,7 +19,11 @@ export default defineComponent({
 	},
 	setup(props, { emit }) {
 		return {
-			confirmDataTest: computed(() => `${props.dataTest}.confirm`),
+			buttonProps: computed(() => ({
+				['data-test']: `${props.dataTest}.confirm`,
```

Comment:
If it's missing dataTest and outside a KtForm, it'll result in `data-test=".confirm"`:
```suggestion
				['data-test']: props.dataTest ? `${props.dataTest}.confirm` : undefined,
```

Reply:
@santiagoballadares fixed